### PR TITLE
requirements changed for sd-core

### PR DESCRIPTION
**Pull request by ancafarcas**

### CODE REVIEW

**Comment by ancafarcas** [comment]
The changes made in sd-core: https://github.com/liveblog/superdesk-core/pull/1

File: client/package.json [hunk 1]
```diff
@@ -45,6 +45,7 @@
     "load-grunt-config": "0.17.1",
     "load-grunt-tasks": "3.2.0",
     "lodash": "2.4.2",
+    "phantomjs": "^2.1.3",
```
Comment (nistormihai) [inline client/package.json]: Why is needed phantomjs?

**Comment by nistormihai** [comment]
beside that comment all LGTM

File: client/package.json (hunk 1 above)
Comment (ancafarcas) [inline client/package.json]: Without this dependency, the branch was not deployed in Bamboo. It kept showing me this error: "UNMET PEER DEPENDENCY phantomjs@>=1.9"

**Comment by nistormihai** [comment]
LGTM

**Pull request merged**